src/GurobiSolverInterface.jl

```diff
@@ -48,6 +48,37 @@ ConicModel(s::GurobiSolver) = LPQPtoConicBridge(LinearQuadraticModel(s))
 
 supportedcones(::GurobiSolver) = [:Free,:Zero,:NonNeg,:NonPos,:SOC,:SOCRotated]
 
+function setparameters!(s::GurobiSolver; mpboptions...)
+    opts = collect(Any,s.options)
+    for (optname, optval) in mpboptions
+        if optname == :TimeLimit
+            push!(opts, (:TimeLimit,optval))
+        elseif optname == :Silent
+            if optval == true
+                push!(opts, (:OutputFlag,0))
```

Comment:
Not sure whether this should be `OutputFlag` or `LogToConsole`.
https://www.gurobi.com/documentation/6.5/refman/logtoconsole.html